server/sonar-server/src/main/java/org/sonar/server/es/EsClient.java

```diff
@@ -166,6 +169,10 @@ public BulkRequestBuilder prepareBulk() {
     return new ProxyBulkRequestBuilder(nativeClient());
   }
 
+  public Builder prepareBulkProcessor(Listener listener) {
```

Comment:
This method does not aim to be used outside `BulkIndexer`. It conflicts with `prepareBulk()`. For this reason I suggest to move it to the only usage of it: `BulkIndexer`.